### ISSUE TITLE
Instructions unclear

### ISSUE THREAD

**Issue by albatorsk** [kayjei/homeassistant-myuplink#6]
First of all; thanks for writing a myUplink integration for HA!

I'd like to try it out but it's not obvious how to set it up.

Step 1 and 3 was easy enough, but already at step 2 it's tricky. How does one create API credentials? By creating an app? If so, does a callback URL need to be specified? Do the api_key and api_token correspond to "client identifier" and "client secret"? Which is which?

So far have not had any luck getting any new entities in HA.

**Comment by kayjei** [comment]
Yes, the way this API is constructed today, you need to create an application. Callback URL will not be needed, this component will poll only.
I have updated the docs with CLIENT_IDENTIFIER and CLIENT_SECRET

If still no luck, you can verify your credentials in their swagger https://api.myuplink.com/swagger/index.html
Here you can authorize and verify different API-calls.

**Comment by albatorsk** [comment]
Thank you! I have verified that my credentials (client_id and client_secret) work using the Swagger link you provided. I've tried making some calls to see what's in the response, for example "GET /v2/systems/me". Using the info from that I can do the "GET /v2/systems/{deviceId}" or "GET /v2/systems/{deviceId}/points" calls, and the responses of those seems fine, returning my heat pump settings and information. *connectionState* is "Connected" and *serialNumber* is the serial number on my heat pump, so I know it's not just example data.

Still no new entities showing up in HA, or maybe I'm just not finding them?

I have an F-series pump that was just recently migrated from the old Nibe Uplink to myUplink. Maybe that's the problem?

Looking in home-assistant.log, I can see the integration loading:
2023-01-02 08:50:14.249 WARNING (SyncWorker_4) [homeassistant.loader] We found a custom integration myuplink which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant

There are no errors in the log regarding this integration.

**Comment by picassoruiz** [comment]
Hi @kayjei 
at first - thanks for Your work on it.

I have unfortunatelly the same issue as @albatorsk - I created everything on dev.myuplink.com and tested on the swagger side (it's working).  Then I installed thru HACS Your integration, put the credentials on configuration.yaml (then restarted ;-) ) and now I cannot find any additional entities. 

I have an S1266

Can You help me out what I did wrong? Or which step I missed?

Cheers
Pawel
